src/runtime/zarf.yaml

```diff
@@ -15,6 +15,8 @@ variables:
 components:
   - name: uds-runtime
     required: false
```

Comment:
Could change at this layer too - shouldn't matter but good to keep aligned.

Reply:
done!